src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -327,36 +493,25 @@ Convert an OpSum object `os` to a TreeTensorNetwork, with indices given by `site
 """
 function TTN(
   os::OpSum,
-  sites::IndsNetwork{V,<:Index};
-  root_vertex::V=default_root_vertex(sites),
+  sites::IndsNetwork;
+  root_vertex=default_root_vertex(sites),
   splitblocks=false,
+  method=:svd,
```

Comment:
We have generally been using `algorithm="svd"` for this kind of thing, best to follow that style/convention.